### PR TITLE
add an improved readme for the project

### PR DESCRIPTION
**Pull request by categulario**
It contains development directions that could be useful for somebody

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,21 +12,38 @@ PDF-Shuffler is written in Python using PyGTK. It is released under the GNU GPL-
 
 In order to install run:
 
- python setup.py install
+    python setup.py install
```
Comment (tgbugs) [inline README.md:15]:
```suggestion
    `python setup.py install`
```

File: README.md [hunk 2]
```diff
@@ -13,21 +12,38 @@ PDF-Shuffler is written in Python using PyGTK. It is released under the GNU GPL-
...
--------------
+## Known issues:
 
 * High memory consumption.
 
 * No autoscrolling during navigation with the keyboard arrows.
 
+## Development
+
+You'll need this libraries installed in your system:
```
Comment (tgbugs) [inline README.md:33]:
```suggestion
You will need these libraries installed in your system:
```

**Review by tgbugs — COMMENTED** [review]
This time around I managed to find https://github.com/logari81/pdfshuffler which I did not find when I was searching before. Should probably update/point to @logari81's version, but not sure how to do that beyond updating the forked from to also note/include that repo. There is also https://github.com/jeromerobert/pdfarranger found in the issues listed on that repo.

**Comment by tgbugs** [comment]
I suggest that I archive this repo and we move over to https://github.com/jeromerobert/pdfarranger since it has a stronger community. I will update the README to reflect this since it seems that for whatever reason this repo is what shows up in the github search results for pdfshuffler. Thoughts?

**Comment by categulario** [comment]
Amazing, I think that's the most sensible thing to do. Thanks for everything. I'll also swith to that project.

**Pull request closed without merging**